frontend/src/text.json

```diff
@@ -56,6 +56,21 @@
             2
         ]
     },
+
+    "Close Contacts Notice": {
+        "texts": [
+            "By clicking agree you acknowledge that anonymous emails will be sent to the list of close contacts you provided.", 
+            "They will be told they may have been exposed to COVID-19 and given resources on what to do.", 
+            "We will never share your identity or the email of the person you have listed."
+        ],
+        "listIndex": [
+            1
+        ],
+        "linkIndex": [
+            2
+        ]
```

Comment:
considering `closeContactNotice.listIndex` & `.linkIndex` are never used, there is any sense on specifying them?